### PR TITLE
Add sticky table headers so it's easier to see which state's data you're looking at 

### PR DESCRIPTION
**Pull request by NikilXYZ**
Added some sticky headers so that it's easy to tell what state you're looking at as you scroll down.
Size is calculated based on table padding, font-size and linespacing, and the border-width.

### CODE REVIEW

**Comment by alex** [comment]
This looks like it has a bad merge conflict or something, because it seems to contain some unrelated changes.

**Comment by NikilXYZ** [comment]
image

I'm not sure what merge conflicts you are seeing?

**Comment by NikilXYZ** [comment]
Strange - I am not sure what all of those extra lines came from - must have been that the lines moved down. Does this need to be rebased?

I can just try another request with those 2 lines

File: battleground-state-changes.html.tmpl [hunk 1]
```diff
@@ -65,11 +75,6 @@
         This website is <a href="https://github.com/alex/nyt-2020-election-scraper">open source</a> and was written by <a href="https://github.com/alex/nyt-2020-election-scraper/graphs/contributors">these people</a>.
     </p>
```
Comment (michaelkirk) [inline battleground-state-changes.html.tmpl:77]: @NikilXYZ - did you intend to delete the following stuff around table shrinking?

**Comment by alex** [comment]
#72 superceeded thi

File: battleground-state-changes.html.tmpl (hunk 1 above)
Comment (michaelkirk) [reply battleground-state-changes.html.tmpl:77]: whoop - disregard my comment. Looks like it was just caught in a rebase.

**Comment by NikilXYZ** [comment]
Perfect. That's what I was wondering

**Pull request closed without merging**